<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>899</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sherd Match Assistance</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>656</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="selectContextLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="hemisphere_l">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hemisphere</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="zone_l">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zone</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="easting_l">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Easting</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="northing_l">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Northing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="hemisphere_cb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="zone_cb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="easting_cb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="northing_cb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="context_cb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="context_l">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Context</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="contextDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>641</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>810</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="update_container">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>1051</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="update_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_36">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>90</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>3</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Batch:</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_37">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>110</y>
       <width>91</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>3</y>
        <width>81</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Piece:</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_38">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>110</y>
       <width>161</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>141</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New Value</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_39">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Find #</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="selected_find_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_40">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>3</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_41">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>91</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>3</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Batch:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="current_batch">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>3</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Piece:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="current_piece">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>10</y>
       <width>161</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color : white</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>141</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Current Value</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="new_piece">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="new_batch">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>120</y>
       <width>30</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="remove_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Remove match</string>
     </property>
    </widget>
    <widget class="QLabel" name="selected_find">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1071</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="finds_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>391</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>258</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="findPhotosLayout">
      <item>
       <widget class="QLabel" name="findFrontPhoto_l">
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="findBackPhoto_l">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>231</width>
       <height>390</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Order by Batch #</string>
      </attribute>
      <widget class="QTreeView" name="modelList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>360</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Order by Area Similarity</string>
      </attribute>
      <widget class="QTreeView" name="sorted_model_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>360</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="model">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>430</width>
       <height>390</height>
      </rect>
     </property>
     <property name="text">
      <string>3dmodel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionSelect_folder">
   <property name="text">
    <string>Select folder</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAdjust_parameters_weights">
   <property name="text">
    <string>Adjust parameters weights</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
